docs/pages/components/panel.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Panel
+id: expand-panel
+keywords: panel
+sidebar: left-navigation-sidebar
+toc: false
+permalink: components/panel.html
+folder: components
+---
+The panel is a container for grouping and displaying information. It can be collapsed to save space on the screen.
+{: .docs-intro}
+
+
+Use the panel if:
+<ul> 
+    <li>You need to group or display information.</li>
+    <li>You want to give users the option of hiding this information.</li>
+    <li>You want to show additional information on demand (for example, a panel could show optional input fields for an advanced search).</li>
+</ul>
+Do not use the panel if:
+<ul>
+    <li>You are designing an object page. Never use panels in the object page content area.</li>
+</ul>
+
+
+## Expandable Panel
+Expandable panels are much like fixed panels, except their content can be expanded and collapsed (including the info toolbar, if available).
+{% capture expand-panel %}
+<div class="fd-expand-panel">
+    <div class="fd-expand-panel__header">
+        <div class="fd-expand-panel__expand">
+            <button class="fd-button fd-button--transparent fd-expand-panel__button" aria-expanded="false" aria-haspopup="true" aria-controls="jhqDKYdf"></button>
```

Comment:
All of these "expand/collapse" buttons need to have a label. It seems best to add an `id` to the Panel Header, and an `aria-labelledby` of the `button` set to the `id` of the header.